### ISSUE TITLE
After add new language, please set the value to default string

### ISSUE THREAD

**Issue by zxl777** [willpowell8/LocalizationKit_iOS#28]
After add new language, Now the value is empty , So app show nothing or just show key.

I think if value is empty,  App should show the defallt value (English) or not key or empty.

BTW:  LocalizationKit_iOS is SO COOL! Thank you.

**Comment by willpowell8** [comment]
Yes this is a good suggestion. I am currently working on the web ui to update it so live editing is not working on 3.X and 4.X versions, but this will be released in the next week.

@zxl777 if you have any more suggestions for features or things that would make things easier please message me directly. Would really like comments.

**Comment by zxl777** [comment]
Where to message you directly ? GITTER?

**Comment by dankrad** [comment]
This would be great :)

**Comment by willpowell8** [comment]
if you want to message me use willpowell8 [at] gmail.com

**Comment by iPhoNewsRO** [comment]
any update about this? or some "import" feature in web UI?
